rascal2/ui/view.py

```diff
@@ -21,12 +23,7 @@ def __init__(self):
         self.undo_view.setWindowIcon(window_icon)
         self.undo_view.setAttribute(QtCore.Qt.WidgetAttribute.WA_QuitOnClose, False)
 
-        self.mdi = QtWidgets.QMdiArea()
-        # TODO replace the widgets below
-        self.plotting_widget = QtWidgets.QWidget()
-        self.terminal_widget = QtWidgets.QWidget()
-        self.controls_widget = QtWidgets.QWidget()
-        self.project_widget = QtWidgets.QWidget()
+        self.initializeMDI()
```

Comment:
note that if you create a new project, MDI is initialised twice: once here and once in `createNewProject`